internal/corazawaf/body_buffer.go

```diff
@@ -60,6 +72,9 @@ func (br *BodyBuffer) Write(data []byte) (n int, err error) {
 			br.length = br.options.MemoryLimit
 			return br.buffer.Write(data[:maxWritingDataLen])
```

Comment:
We probably want to handle gracefully the panic here and return an error.

Reply:
You are definitely right, it makes homogeneous the approach between Go and TinyGo